app/routes/install/index.js

```diff
@@ -72,7 +75,8 @@ router.get('/razeedeploy-job', asyncHandler(async (req, res, next) => {
     const view = {
       NAMESPACE: req.query.namespace || 'razeedeploy',
       COMMAND: req.query.command || 'install',
-      ARGS_ARRAY: args_array
+      ARGS_ARRAY: args_array,
+      UUID: 'razeedeploy-job-' + time.toString()
```

Comment:
```suggestion
      UUID: `razeedeploy-job-${Date.now()}`
```

Reply:
the job already has the name defined, we just need the `-` and the time added to the name.